README.md

```diff
@@ -82,6 +82,48 @@ Definitions of the flags used in the below example
 swagger-codegen generate -l rails5 -i public/doc/swagger-2.yaml -o /tmp/sp -t swagger-codegen-templates/'
 ```
 
+## Testing with Dev Insights UI
+
+If you want to run the service portal locally on your dev machine but access the UI on the dev cluster you would need to do the following
+
+Install Docker in your machine
+
+Clone the following repo to your dev machine
+
+[Insights Proxy](https://github.com/RedHatInsights/insights-proxy)
+
+Follow the instructions in [README.md](https://github.com/RedHatInsights/insights-proxy/blob/master/README.md)
+
+
+In the service-portal repository the dev_configs directory contains 2 Java script files which can be used with insights proxy to route the service portal requests to your dev machine
```

Comment:
2 javascript files

Reply:
@bzwei The insights proxy uses java script for its configuration which is attached in this PR as new files.

Reply:
@bzwei is pointing out that the files are for the `JavaScript` (one word) language. https://en.wikipedia.org/wiki/JavaScript